packages/11ty/_plugins/transforms/outputs/epub/transform.js

```diff
@@ -31,8 +31,8 @@ module.exports = function(eleventyConfig, collections, content) {
     images.forEach((img) => {
       const src = img.getAttribute('src')
       if (!src) return
-      const match = (`(^${imageDir}|^\/${iiifOutputDir})`)
-      const regex = new RegExp(match, 'g')
+      const pattern = `^(${imageDir}|/${iiifOutputDir})`
```

Comment:
```suggestion
      const pattern = `^(${imageDir}|\/${iiifOutputDir})`
```
does the `/` not need to be escaped?

Reply:
It does not.